layouts/partials/list.html

```diff
@@ -0,0 +1,150 @@
+            {{ end }}
+            <div id="pageTurnSpacer" style="display:inline; margin: 0px {{ $sideMargin }}"></div>            
+              {{ with $currentPage.PrevInSection }}
+              <a id="nextLink" href="{{ .RelPermalink }}" style="font-size: {{ $fontSize }}">Next - {{ .Name }}</a>                       
+              {{ end }}
+            </div> 
+            <!-- Use (1) or do not use (0) JavaScript to correct links. -->
+            <div id="useJS" style="display: none">0</div>
+          {{ end }}            
+        {{ end }}
+      {{ end }}      
+    {{ end }}
+  {{ else }}
+    {{ $extractParent := $currentPage.Parent.Page }}
+    {{ $currentParent := (replaceRE "Page\\(\\/(.*)\\)" "$1" $extractParent) }}    
+    {{ if eq $currentParent (.Parent) }}              
+      {{ if .Weight }}
+        <!-- Add content to be modified by JavaScript. -->
+        {{ if $currentPage.IsMenuCurrent "docs" . }}
+          <!-- Add filler html and use JavaScript to put in correct links. -->
+          <div style="margin: {{ $verticalMargin }} 0px">            
+            <a id="previousLink" href="javascript:void(0)" style="font-size: {{ $fontSize }}">Previous - Page</a>
+            <div id="pageTurnSpacer" style="display:inline; margin: 0px {{ $sideMargin }}"></div>            
+            <a id="nextLink" href="javascript:void(0)" style="font-size: {{ $fontSize }}">Next - Page</a>
+          </div>
+          <!-- Use (1) or do not use (0) JavaScript to correct links. -->
+          <div id="useJS" style="display: none">1</div>
+        {{ end }}            
+      {{ else }}
+        {{ if $currentPage.IsMenuCurrent "docs" . }}
+          <!-- Add content as structured by Hugo -->
+          <div style="margin: {{ $verticalMargin }} 0px">      
+            {{ with $currentPage.NextInSection }}
+              <a id="previousLink" href="{{ .RelPermalink }}" style="font-size: {{ $fontSize }}">Previous - {{ .Name }}</a>            
+            {{ end }}
+            <div id="pageTurnSpacer" style="display:inline; margin: 0px {{ $sideMargin }}"></div>            
+            {{ with $currentPage.PrevInSection }}
+              <a id="nextLink" href="{{ .RelPermalink }}" style="font-size: {{ $fontSize }}">Next - {{ .Name }}</a>                       
+            {{ end }}
+          </div> 
+          <!-- Use (1) or do not use (0) JavaScript to correct links. -->
+          <div id="useJS" style="display: none">0</div>
+        {{ end }}            
+      {{ end }}
+    {{ end }}      
+  {{ end }}
+{{ end }}
+                  
+<script>
+ // The ids used in doc page nav at bottom.
+ var previousLink = document.getElementById("previousLink");
+ var nextLink = document.getElementById("nextLink");
+ var pageTurnSpacer = document.getElementById("pageTurnSpacer");
+ var useJS = document.getElementById("useJS");
+ 
+ // Only run statement to add correct links if html id useJS innerHTLM is 1.

```

Comment:
Using html from Hugo build (_id="useJS"_) for variable that tells if content is genereated from Hugo (_alphabetical sorted pages_) or needs links and text to be changed with JavaScript.